src/System.Security.Cryptography.Xml/src/System/Security/Cryptography/Xml/KeyInfoX509Data.cs

```diff
@@ -60,10 +60,10 @@ public KeyInfoX509Data(X509Certificate cert, X509IncludeOption includeOption)
                     // Can't honor the option if we only have a partial chain.
                     if ((chain.ChainStatus.Length > 0) &&
                         ((chain.ChainStatus[0].Status & X509ChainStatusFlags.PartialChain) == X509ChainStatusFlags.PartialChain))
-                        throw new CryptographicException(CAPI.CERT_E_CHAINING);
+                    throw new CryptographicException(unchecked((int)Interop.Crypt32.CertChainPolicyErrors.CERT_E_CHAINING));
```

Comment:
Ok - I'll add a string resource for the message.

Is  "An internal certificate chaining error has occurred." acceptable? That's what it says in [netfx](https://referencesource.microsoft.com/#System.Security/system/security/cryptography/cryptoapi.cs,4d223587f5caffb6,references).